src/all.js

```diff
@@ -11,6 +11,7 @@ import Tabs from './components/tabs/tabs'
 import SdnHeader from './components/_custom/header/header'
 import SdnTimeline from './components/_custom/timeline/timeline'
 import SdnAppearLink from './utilities/appear-link/appear-link'
+import SdnInputRodneCislo from './components/_custom/input-rodne-cislo/intput-rodne-cislo'
```

Comment:
Ideme po slovensky? Nie som uplne proti len sa pytam.

Reply:
Kedze to je forknute ja by som bol za en :)
Inac tu budes mat taky misung medzi en/sk a bude sa to strasne zle citat. Nehovoriac ze bez diakritiky niektore veci mozu menit vyznam.

Reply:
Cast custom veci je uz slovenskych a cast anglickych. Problem s anglickymi mam v tom, ze na jeden slovensky termin existuje viacero podobne dobrych prekladov, a teda pri preklade sa trochu straca informacia. Napr. rodne cislo -> identification number -> identfikacne cislo (rodne cislo / pas / cislo vodicskeho / BIFO /... ). Ak sa dohodneme na anglictine tak to prelozim.